docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -614,10 +614,10 @@ performance and robustness reasons.
 The format of the name is the same as when passed to `podman pull`. So, it supports using
 `:tag` or digests to guarantee the specific image version.
 
-As a special case, if the `name` of the image ends with `.image`, Quadlet will use the image
-pulled by the corresponding `.image` file, and the generated systemd service contains
-a dependency on the `$name-image.service`.
-Note that the corresponding `.image` file must exist.
+Special Cases:
+
+* If the `name` of the image ends with `.image`, Quadlet will use the image pulled by the corresponding `.image` file, and the generated systemd service contains a dependency on the `$name-image.service`. Note that the corresponding `.image` file must exist.
```

Comment:
Since Quadlet now allows setting the name of the generated service, it should be something like:

"a dependency on the `$name-image.service` (or the service name set in the `.image` file)"

The same applies for the `.build` file and other places in this change